projects/plugins/jetpack/extensions/shared/components/product-management-controls/inspector-control.js

```diff
@@ -88,9 +85,9 @@ export default function ProductManagementInspectorControl() {
 		if ( isCustomTitle ) {
 			return;
 		}
-		setTitle( getTitleByProps( isMarkedAsDonation, interval ) );
+		setTitle( getTitleByProps( false, interval ) );
```

Comment:
```
getTitleByProps( false, interval )
```
I believe we could remove this first parameter from the method as it will always be false...

Reply:
I've removed this.